Formula/openjdk.rb

```diff
@@ -39,8 +39,8 @@ class Openjdk < Formula
   resource "boot-jdk" do
     on_macos do
       if Hardware::CPU.arm?
-        url "https://download.java.net/java/early_access/jdk16/31/GPL/openjdk-16-ea+31_osx-x64_bin.tar.gz"
-        sha256 "8e4a8fdd2d965067bdb56e1a72c7c72343d571b371ac61eee74d9e71bbef63e8"
+        url "https://cdn.azul.com/zulu/bin/zulu16.30.15-ca-jdk16.0.1-macosx_aarch64.tar.gz"
+        sha256 "47f50e9c120130a77a77e65ccdb4ca4e101fe662bb429ba95668811c8618ab67"
```

Comment:
That's not the same as OpenJDK though. We'll need to find an OpenJDK bootstrap or risk getting a bunch of weird bugs that only happen on ARM

Reply:
Zulu JDK 7 is already used as boot JDK in `openjdk@8` for Linux and macOS, so I don't think that this is an issue. The old snapshot is unreachable anyway. Oracle used to provide some pre-release arm64 snapshots of v16 before it's GA, but then they decided to ship macOS@ARM64 port it only in version 17. See [JEP 391](https://openjdk.java.net/jeps/391). I actually updated a link to the latest material for this JEP.